tests/test_icml_conference.py

```diff
@@ -3830,6 +3831,97 @@ def test_comment_stage(self, openreview_client, helpers):
+            reply_type=openreview.stages.CustomStage.ReplyType.REPLY,
+            invitees=[openreview.stages.CustomStage.Participants.AUTHORS, openreview.stages.CustomStage.Participants.AREA_CHAIRS_ASSIGNED],
+            readers=[openreview.stages.CustomStage.Participants.SENIOR_AREA_CHAIRS_ASSIGNED,openreview.stages.CustomStage.Participants.AREA_CHAIRS_ASSIGNED,openreview.stages.CustomStage.Participants.AUTHORS],
+            start_date=start_date,
+            due_date=end_date,
+            content={
+                'title': {
+                    'order': 1,
+                    'description': '(Optional) Brief summary of your comment.',
+                    'value': {
+                        'param': {
+                            'type': 'string',
+                            'maxLength': 500,
+                            'optional': True,
+                            'deletable': True
+                        }
+                    }
+                },
+                'comment': {
+                    'order': 2,
+                    'description': 'Your comment or reply (max 5000 characters). Add formatting using Markdown and formulas using LaTeX. For more information see https://openreview.net/faq',
+                    'value': {
+                        'param': {
+                            'type': 'string',
+                            'maxLength': 5000,
+                            'markdown': True,
+                            'input': 'textarea'
+                        }
+                    }
+                }
+            },
+            multi_reply=True
+        )
+        venue.create_custom_stage()
+        helpers.await_queue_edit(openreview_client, 'ICML.cc/2023/Conference/-/Author_AC_Confidential_Comment-0-1', count=1)
+
+        assert len(openreview_client.get_invitations(invitation='ICML.cc/2023/Conference/-/Author_AC_Confidential_Comment')) == 100
+        invitation = openreview_client.get_invitation('ICML.cc/2023/Conference/Submission1/-/Author_AC_Confidential_Comment')
+        assert invitation.invitees == [
+            'ICML.cc/2023/Conference/Program_Chairs',
+            'ICML.cc/2023/Conference/Submission1/Area_Chairs',
+            'ICML.cc/2023/Conference/Submission1/Authors'
+        ]
+
+        author_client = openreview.api.OpenReviewClient(username='peter@mail.com', password=helpers.strong_password)
+        confidential_comment_edit = author_client.post_note_edit(
+            invitation='ICML.cc/2023/Conference/Submission1/-/Author_AC_Confidential_Comment',
+            signatures=['ICML.cc/2023/Conference/Submission1/Authors'],
+            note=openreview.api.Note(
+                replyto=submissions[0].id,
+                content={
+                    'comment': { 'value': 'Author confidential comment to AC' },
+                }
+            )
+        )
+        helpers.await_queue_edit(openreview_client, edit_id=confidential_comment_edit['id'])
```

Comment:
could you add a test to verify the readers are getting notified of the new confidential comment?

Reply:
I noticed that PCs and the assigned SAC weren't notified even though they're readers. I think it's because `email_sacs` and `email_pcs` are false by default. I'm guessing this is how it's supposed to function?

Reply:
Yes, usually PCs and SACs don't receive emails by default.